### PR TITLE
Build js packages at code-generation time

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Build Javascript packages after code generation no longer manipulate or require workspaces.

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -223,6 +223,11 @@ alias(
     actual = "@nodejs//:yarn",
 )
 
+alias(
+    name = "node",
```
Comment (shayne-fletcher) [inline BUILD]: As it turns out, this is not required. Doesn't hurt though.

File: language-support/ts/codegen/src/TsCodeGenMain.hs [hunk 2]
```diff
@@ -51,13 +52,14 @@ configConsts :: T.Text -> ConfigConsts
 configConsts sdkVersion = ConfigConsts
   { pkgDependencies = HMS.fromList
       [ (NpmPackageName "@mojotech/json-type-validation", NpmPackageVersion "^3.1.0")
-      , (NpmPackageName "@daml/types", NpmPackageVersion sdkVersion)
+      , (NpmPackageName "@daml/types"
```
Comment (shayne-fletcher) [inline language-support/ts/codegen/src/TsCodeGenMain.hs]: We could do something with the resolutions field in the daml2ts package.json if that's considered preferable.

File: BUILD (hunk 1 above)
Comment (shayne-fletcher) [reply BUILD]: Removed. Not used, breaks CI.

File: language-support/ts/codegen/src/TsCodeGenMain.hs (hunk 2 above)
Comment (hurryabit) [reply language-support/ts/codegen/src/TsCodeGenMain.hs]: Yes, I'd be more happy if `daml2ts` wouldn't know about this and the actual tests do the "rerouting". Doesn't https://github.com/digital-asset/daml/blob/master/language-support/ts/codegen/tests/ts/package.json do this already anyways?

**Pull request closed without merging**